faqs/stocks/which-screener.md

```diff
@@ -9,75 +9,122 @@ description: >-
 
 ## Introduction
 
-Even though screeners are useful in finding out the investment opportunities which suits your criteria and to monitor the existing investment, it might not be a good idea to take a screener at its face value.
+Though stock-screeners calculate a lot of things, you must do you own calculations too. You will miss a lot of good opportunities or make some expensive mistakes if you consider only the pre-calculated values.
 
-You might see an opportunity or a red-flag from these screeners. However, it could easily just be a calculation error, or usage of incorrect formula, or not removing an outlier.
+The objective of this article is not to demean the websites but to show the importance of your own calculations.
 
-This article is to sensitize you that you might have to do more research rather than just going with the presented data from the UI of a screener.
+----
 
-## Examples of incorrect data in screeners
+## Examples of dividend yield
 
-We've come across some incorrect computations and results that are outright wrong. We've also informed the teams behind these screeners. At the time of writing this, these have not been addressed.  
-  
-It's our duty to warn others to be mindful of these, while they embark on their research.
+Lets consider the dividend yield of Britannia Industries as on 29th April 2021:
```

Comment:
We should probably expand on this a bit. If a user is reading this on 31st December 2022, what can they do to obtain the same data, for these fields?

Maybe we can add a note / info panel on that.

Reply:
Also, when we write Britannia Industries, could we add the ticker symbol, so as to avoid any ambiguity for a reader?

Reply:
Hey, good idea. I have fixed these in https://github.com/indiainvestments/content/commit/de8441f26bc353c55bde01d151670dcc42a7b555

 included a web-archive page for websites that supported it. In other cases I have included screenshots.

I have kept all these copies together as a footnote. This should make the whole thing complete yet concise.